### PR TITLE
add class TransactionId

### PR DESCRIPTION
**Pull request by sergiipopil**
method to Get TransactionData

### CODE REVIEW

File: Classes/PaymentInfo.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```
Comment (MarkIlchenko) [inline Classes/PaymentInfo.cs:4]: Good!

File: Classes/PaymentInfo.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+namespace Task_PR_Team1.Classes
+{
+    internal class PaymentInfo
+    {
+        private int _IBAN;
+        private int _transactionId;
+
+        public int IBAN { get { return _IBAN;} set { _IBAN = value; } }
+        public int TransactionId { get { return _transactionId; } set { _transactionId = value; } }
+
```
Comment (MarkIlchenko) [inline Classes/PaymentInfo.cs:16]: Good!

**Review by MarkIlchenko — APPROVED** [review]

**Review by Boichenko1988 — COMMENTED** [review]
Looks fine

**Review by Boichenko1988 — APPROVED** [review]
Approve

**Pull request merged**